build.gradle

```diff
@@ -240,6 +240,15 @@ model {
                 }
             }
         }
+        compileCpp(Task) {
+            $.binaries.each { binary ->
```

Comment:
What's the `$` mean in groovy?

Reply:
`$` is a gradle-specific symbol, it means model root.

Reply:
Thanks!